capstone/src/main/webapp/script.js

```diff
@@ -159,3 +159,13 @@ function displayPanel(name) {
   document.getElementById("panel").style.display = "block";
   document.getElementById("restaurant-info").innerHTML = name;
 }
+
+function getSmallBizNames() {
+  fetch('/gimmeSmallBizNames').then(response => response.json()).then((restaurantNames) => {
```

Comment:
lol I don't want to crush your creativity but I think it's better to make the fetching as neutral as possible and to use common lingo. In google if you are making an API call we use 'fetch' as a prefix. So in this case `fetchPlaceNames` or `fetchBusinessNames` would be more appropriate.

Reply:
fair enough